hostkeys.sls

```diff
@@ -0,0 +1,21 @@
+{% for keypair in salt['pillar.get']('sshd:hostkeys') %}
```

Comment:
I think it's best to use the actual filenames in the pillar directly, as it has a little less overhead. 

The only differences between private and public keys (deployment wise) is the permissions. I'd set it to `0600` by default and to `0640` if the filename ends with `.pub`. 

Pillar structure would then be looking like this:

```yaml
sshd:
  hostkeys:
    ssh_host_ed25519_key: |
      ...
    ssh_host_ed25519_key.pub: |
      ...
```

Reply:
Ok, I'll give that a shot. Note however that OpenSSH default for pub keys appears to be `0644`. Should I still go with `0640` or stick with the default?